### PR TITLE
deploy-application-to-heroku

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: ejs-templating/deploy-application-to-heroku/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+Currently, your application works in your local environment. But it's time to show the whole world what it can do.
+
+In this section, we will be looking at deploying our Rails application code to Heroku. Heroku is a PaaS that lets companies build, deliver, monitor and scale apps.
```
Comment (avishekjana) [inline ejs-templating/deploy-application-to-heroku/README.md:3]: In this section, we will be looking at deploying our Node.Js application code to Heroku. Heroku is a PaaS that lets companies build, deliver, monitor and scale apps.

File: ejs-templating/deploy-application-to-heroku/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+It will also [add a new _remote_](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) to your Git configuration, so you can push your code to Heroku.
+
+5. Since Heroku runs `npm start` by default for node projects, you’ll have to define that script in your `package.json`.
+```js
+"scripts": {
+   "start": "node app.js",
+   "test": "echo \"Error: no test specified\" && exit 1"
+}
+```
+Update your scripts attribute in `package.json` to the above command. Basically, this command tells Heroku to run the command node app.js, which is the same command you’d use to run a node app on your terminal.
```
Comment (avishekjana) [inline ejs-templating/deploy-application-to-heroku/README.md:31]: Update your scripts attribute in `package.json` to the above command. Basically, this command tells Heroku to run the command `node app.js`, which is the same command you’d use to run a node app on your terminal.

File: ejs-templating/deploy-application-to-heroku/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+6. Next, we need our app to connect to your **Postgres** database in order for it to show data. To do that, we will start by installing the Heroku Postgres add-on.
+```
+heroku addons:create heroku-postgresql:hobby-dev
+```
+You can check if it installed successfully by going to your Heroku Dashboard and checking your app’s add-ons.
```
Comment (avishekjana) [inline ejs-templating/deploy-application-to-heroku/README.md:37]: You can check if it is installed successfully, by going to your Heroku Dashboard and checking your app’s add-ons.

**Review by avishekjana — CHANGES_REQUESTED** [review]

File: ejs-templating/deploy-application-to-heroku/README.md (hunk 1 above)
Comment (vikispeaks) [reply ejs-templating/deploy-application-to-heroku/README.md:3]: Done.

File: ejs-templating/deploy-application-to-heroku/README.md (hunk 2 above)
Comment (vikispeaks) [reply ejs-templating/deploy-application-to-heroku/README.md:31]: Done.

File: ejs-templating/deploy-application-to-heroku/README.md (hunk 3 above)
Comment (vikispeaks) [reply ejs-templating/deploy-application-to-heroku/README.md:37]: Done.

**Pull request merged**